src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -129,23 +130,60 @@ public void getTableDataWithNotExistingTable() throws SQLException {
 
     @Test
     public void getTableDataWithValidTableTwoRows() throws SQLException {
-        executeSqlQuery("CREATE TABLE test_table(" +
-                "id INTEGER," +
-                "name VARCHAR(128)" +
-                ")");
-        executeSqlQuery("INSERT INTO test_table VALUES(1,'name1')");
-        executeSqlQuery("INSERT INTO test_table VALUES(2,'name2')");
+        createTestTableWithIdAndName(TEST_TABLE_NAME);
+        executeSqlQuery("INSERT INTO " + TEST_TABLE_NAME + " VALUES(1,'name1')");
+        executeSqlQuery("INSERT INTO " + TEST_TABLE_NAME + " VALUES(2,'name2')");
         DataSet row1 = new DataSet(2);
         row1.insertValue(0, "1");
         row1.insertValue(1, "name1");
         DataSet row2 = new DataSet(2);
         row2.insertValue(0, "2");
         row2.insertValue(1, "name2");
         DataSet[] expectedArray = new DataSet[]{row1, row2};
-        DataSet[] actualArray = databaseManager.getTableData("test_table");
+        DataSet[] actualArray = databaseManager.getTableData(TEST_TABLE_NAME);
         assertThat(actualArray, arrayContainingInAnyOrder(expectedArray));
     }
 
+    @Test
+    public void insertIntoTableWithExistingTable() throws SQLException {
```

Comment:
Maybe insertWhenTableExists?

Reply:
I use "with" instead of "when", but it's a good point